CWE/events/italy.txt

```diff
@@ -54,460 +126,341 @@ country_event = {
+		
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = -0.05 value = communist }
+		scaled_consciousness = 2
+		scaled_militancy = 2
 		}
-		set_country_flag = very_radical_ICP
-		ai_chance = { factor = 0.2 }
+      
+		add_country_modifier = { name = political_strife duration = 1825 }
+
+		ai_chance = { factor = 0.01 }
 	}
-	
 }
 
-#Italian Communist Party in 1950s
+#1948 elections
 country_event = {
-  id = 8007738
-  title = EVT_8007738_NAME
-  desc = EVT_8007738_DESC
-  picture = "nwo2_italian_communist_party"
-  fire_only_once = yes
+	id = 8007771
+	title = "$YEAR$ Italian General Election"
+	desc = EVTDESC8007771
+	picture = "ITA_1948_elections"
+	
+	fire_only_once = yes
 
 	trigger = {
 		tag = ITA
-		year = 1950 NOT = { year = 1952 }
-		has_country_flag  = very_moderate_ICP
-		OR = { government = democracy government = hms_government }
-		NOT = { 
-			OR = { 
-				ruling_party_ideology = socialist 
-				ruling_party_ideology = communist 
-				ruling_party_ideology = communist_social 
-			} 
-		}
-		has_global_flag = blocsenabled
+		year = 1948
+		NOT = { year = 1950 }
+		
+		government = democracy
+		war = no
+		is_vassal = no
+	
 	}
 
 	mean_time_to_happen = { months = 2 }
 
 	option = {
-		name = EVT_8007738_A
-		any_pop = { consciousness = 5 }
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = socialist }
+		name = "Centrist Coalition (DC-US-BN-PRI)"
+		ruling_party_ideology = conservative
+		any_pop = { 
+			ideology = { factor = 0.1 value = conservative }
 		}
+		set_country_flag = 1948_ITA_Centrism
 		ai_chance = { factor = 0.7 }
 	}
 
 	option = {
-		name = EVT_8007738_B
-		any_pop = { consciousness = 2.5 }
-		country_event = 8007741 
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
+		name = "Popular Democratic Front (FDP)"
+		ruling_party_ideology = communist
+		any_pop = { 
+			ideology = { factor = 0.2 value = communist }
 		}
-		set_country_flag = radical_ICP
-		ai_chance = { factor = 0.2 }
-	}
-
-	option = {
-		name = EVT_8007738_C
-		any_pop = { militancy = -5 }
-		country_event = 8007742
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = conservative }
+		any_pop = { 
+			ideology = { factor = 0.1 value = socialist }
 		}
-		ai_chance = { factor = 0.1 }
+		set_country_flag = 1948_ITA_FDP
+		ai_chance = { factor = 0.3 }
 	}
 
 }
 
-#Italian Communist Party in 1960s
+#Togliatti assassination attempt
 country_event = {
-  id = 8007739
-  title = EVT_8007739_NAME
-  desc = EVT_8007739_DESC
-  picture = "nwo2_italian_communist_party"
+  id = 8007736
+  title = EVT_8007736_NAME
+  desc = EVT_8007736_DESC
+  picture = "nwo2_palmiro_togliatti"
   fire_only_once = yes
 
 	trigger = {
 		tag = ITA
-		year = 1960 NOT = { year = 1962 }
-		has_country_flag  = very_moderate_ICP
-		OR = { government = democracy government = hms_government }
-		NOT = { 
```

Comment:
The ruling party check should be retained since if Togliatti was PM then he would have better security, preventing the assassination event.

Reply:
The shooting is kinda needed to justify more restrictive laws and whetevr we can use reality to base AltHis it's better, also it wouldn't be that sure he would get better security, many even in positions of power would very much not like him and he had made the Church an enemy which is a big threat in italian politics